Source/CesiumRuntime/Public/CesiumRuntimeSettings.h

```diff
@@ -52,4 +52,18 @@ class CESIUMRUNTIME_API UCesiumRuntimeSettings : public UDeveloperSettings {
    */
   UPROPERTY(Config, EditAnywhere, Category = "Experimental Feature Flags")
   bool EnableExperimentalOcclusionCullingFeature = false;
+
+  /**
+   * The number of requests to handle before each prune of old cached results
+   * from the database. Must restart Unreal for changes to take effect.
+   */
+  UPROPERTY(Config, EditAnywhere, Category = "Cache")
```

Comment:
There's a meta attribute you can add to a property to tell the Editor it needs to restart when the property changes.
```suggestion
  UPROPERTY(Config, EditAnywhere, Category = "Cache", meta = (ConfigRestartRequired = true))
```